lib/api/api.dart

```diff
@@ -520,6 +523,29 @@ class _APIRequests {
     await _actionExecution(APIAction.GET_HOMEWORK);
     http.postToAPI("/homework/v1/report/$id", {}, jsonEncode({}), _api._authenticationUser.getJWT());
   }
+
+  Future<bool> sendKrankmeldung(String sek, String name, String grade, String klasse, String leader, String email, String remarks) async {
+    await _actionExecution(APIAction.SEND_KRANKMELDUNG);
+    var response = await http.postToAPI(
+        '/krankmeldung',
+        null,
+        jsonEncode({
+          "sek": sek,
+          "name": name,
+          if (sek == 'sek1') "klasse": klasse,
+          if (sek == 'sek1') "klassenleitung": leader,
+          if (sek == 'sek2') "stufe": grade,
+          if (sek == 'sek2') "stufenleitung": leader,
+          "email": email,
+          "bemerkungen": remarks
+        }),
+        null);
+    if (response != null) {
+      final jsonResponse = json.decode(response)['message'];
+      return (jsonResponse == "Krankmeldung erfolgreich abgegeben" ? true : false);
```

Comment:
Das ist doch ziemlich hässlig. Wenn das aktuell so ist. Sollten wir das in der API anpssen, dass auch immer ein success boolean zurückkommt.
Alternativ eben einfach nur 200/201 zulassen (das passiert glaube ich automatisch)

Abgesehen davon kannst `? true : false` natürlich